requirements.txt

```diff
@@ -19,4 +19,3 @@ ninja
 # setuptools was removed from default python install
 setuptools ; python_version >= "3.12"
 packaging
-optree>=0.11.0
```

Comment:
We can't remove it for every other versions, that would be not so good for testing, you can use `; python_version <= "3.12"` to preserve the current behavior for other versions.